supervision/draw/utils.py

```diff
@@ -9,7 +9,7 @@
 
 
 def draw_line(
-    scene: np.ndarray, start: Point, end: Point, color: Color, thickness: int = 2
+    scene: np.ndarray, start: Point, end: Point, color: Color=Color(163, 81, 251), thickness: int = 2
```

Comment:
Please use our color scheme in like "sv.Color.WHITE"  for all them. You can find them in here 

https://github.com/roboflow/supervision/blob/337b0e4a5b5df9617ecd166a008b92472526bd2f/supervision/draw/color.py#L231